index.js

```diff
@@ -531,6 +531,27 @@ exports.generateAddress = function (from, nonce) {
   return exports.rlphash([from, nonce]).slice(-20)
 }
 
+/**
+ * Generates an address for a contract created using CREATE2
+ * @param {Buffer} from the address which is creating this new address
+ * @param {Buffer} salt a salt
+ * @param {Buffer} inputCode the code of the contract being created
+ * @return {Buffer}
+ */
+exports.generateAddress2 = function (from, salt, inputCode) {
+  from = exports.toBuffer(from)
+  salt = exports.toBufer(salt)
+  inputCode = exports.toBuffer(inputCode)
+
+  let address = exports.keccak256(Buffer.concat([
+    Buffer.from('0xff', 'hex'),
```

Comment:
shouldn't `from` be part of the hash?

as mentioned in https://github.com/ethereum/EIPs/pull/1014#issuecomment-412106601

Reply:
Yup.. good call.